srcpkgs/rtl8723bt-firmware/template

```diff
@@ -0,0 +1,27 @@
+# Template file for 'rtl8723bt-firmware'
+pkgname=rtl8723bt-firmware
+version=0.0.202008
+revision=1
+_commit=8840b1052b4ee426f348cb35e4994c5cafc5fbbd
+archs=noarch
```

Comment:
do not add new packages with noarch defined, please

Reply:
gone :+1: